src/Orchard.Cms.Web/Modules/Orchard.Recipes/RecipeSteps/CommandStep.cs

```diff
@@ -35,11 +36,16 @@ public override Task ExecuteAsync(RecipeExecutionContext context)
             foreach(var command in step.Commands)
             {
                 Logger.LogInformation("Executing command: {0}", command);
-
-                var commandParameters = _commandParameterParser.Parse(_commandParser.Parse(command));
-
-                _commandManager.ExecuteAsync(commandParameters);
-
+                using (var memoryStream = new MemoryStream())
+                {
+                    using (var output = new StreamWriter(memoryStream))
```

Comment:
When a command is executed from command line this `commandParameters.Output` is pointing to standard output but when it is executed from a recipe it is null. So, I saw two options: in every command we should check output is not null before writing anything on it or adding an output for commands executed through a recipe. I think it is better we can know how a command executed so by the moment I preferred to provide an output that is written to the logger, but in the future I suppose we can write it in a kind of recipe journal.